manifests/init.pp

```diff
@@ -92,6 +92,9 @@
 # $pulpcore_cache_expires_ttl::                The number of seconds that content should be cached for.
 #                                              Specify 'None' to never expire the cache.
 #
+# $additional_pulpcore_import_paths::          Additional allowed paths that Pulpcore can use for content imports, or sync from using file:// protocol
+# $additional_pulpcore_export_paths::          Additional allowed paths that Pulpcore can use for content exports
```

Comment:
```suggestion
#
# $additional_pulpcore_export_paths::          Additional allowed paths that Pulpcore can use for content exports
```

Reply:
Perhaps this should also follow the theme of prefix? e.g.

```
pulpcore_additional_expore_paths
pulpcore_additional_import_paths
```

Reply:
I missed this in the first round of updates that I pushed. But it's a good point, I'll update it again in a moment.